apps-devstg/us-east-1/k8s-eks-demoapps/addons/addons.tf

```diff
@@ -0,0 +1,55 @@
+################################################################################
+# EKS Addons
+################################################################################
+
+resource "aws_eks_addon" "this" {
+  # Not supported on outposts
+  for_each = { for k, v in local.addons_available : k => v if !try(v.before_compute, false) }
```

Comment:
@juanmatias could you elaborate on why the "before_compute" is needed?

Reply:
"Just in case" is the keyword here. I don't like YAGNI and so I would recommend that we don't keep definitions that we won't be using in the short/mid term.